src/utilities/Libify.js

```diff
@@ -459,6 +459,68 @@ Libify.Operation.revokeSponsorship = function(opts) {
   }
 }
 
+Libify.Operation.clawback = function(opts) {
+  assertNotEmpty(opts.asset, 'Clawback operation requires asset');
+  assertNotEmpty(opts.from, 'Clawback operation requires account from which the asset is clawed back');
+  assertNotEmpty(opts.amount, 'Clawback operation requires amount clawed back');
+  return Sdk.Operation.clawback({
+    asset: Libify.Asset(opts.asset),
+    from: opts.from,
+    amount: opts.amount,
+    source: opts.sourceAccount,
+  })
+}
+
+Libify.Operation.clawbackClaimableBalance = function(opts) {
+  assertNotEmpty(opts.balanceId, 'Clawback Claimable Balance operation requires claimable balance ID');
+
+  return Sdk.Operation.clawbackClaimableBalance({
+    balanceId: opts.balanceId,
+  });
+}
+
+Libify.Operation.setTrustLineFlags = function(opts) {
+  assertNotEmpty(opts.asset, 'Set Trust Line Flags operation requires asset');
+  assertNotEmpty(opts.trustor, 'Set Trust Line Flags operation requires trustor');
+
+  if (
+    !(opts.setFlags && opts.setFlags.length) &&
+    !(opts.clearFlags && opts.clearFlags.length)
```

Comment:
I think we could use optional chaining here to make this a little nicer 
```
!(opts.setFlags?.length)
```

Reply:
didn't know about this JS feature, neat!

Reply:
Keeping it as is for now, until we start using TypeScript.